mlrun/launcher/base.py

```diff
@@ -356,12 +356,12 @@ def _wrap_run_result(
         if result:
             run = mlrun.run.RunObject.from_dict(result)
             logger.info(
-                f"run executed, status={run.status.state}", name=run.metadata.name
+                f"Run executed, status={run.status.state}", name=run.metadata.name
             )
             if run.status.state == "error":
                 if runtime._is_remote and not runtime.is_child:
-                    logger.error(f"runtime error: {run.status.error}")
-                raise mlrun.runtimes.utils.RunError(run.status.error)
+                    logger.error("Runtime error", status=run.status.to_dict())
```

Comment:
is runtime error is the correct wording? I understand that it is in Run time but we also have runtime objects and Run objects. and eventually this is a run error so lets this on another error message